### ISSUE TITLE
Can't add some games

### ISSUE THREAD

**Issue by denny-dee** [moraroy/NonSteamLaunchers-On-Steam-Deck#458]
I don't get why, but NSL won't add "Dragon Age: Origins" and "The Witcher: Enhanced Edition" to my Steam library. I've verified by downloading Flatout that it's still running smoothly and is able to locate new games. These two games may have bugged during the installation, as I seem to have ran out of space when installing them. I've uninstalled both now, re-installed Witcher, but still no luck :c Also, it won't run when adding to Steam library manually with any Proton version. However, I was able to successfully reach the game launcher and menu when started directly from GOG launcher.
I've re-downloaded the script, that didn't do much.
Also, as of yesterday, I have both "Gog Galaxy" and "GOG Galaxy" as separate apps in my library, maybe this is related somehow?

I'm bashing my head, this doesn't make any sense. Any ideas? @moraroy appreciate the app btw, you rock

**Comment by denny-dee** [comment]
@moraroy yeah, maybe it's related to colons, but "Styx: Shards of Darkness" was picked up with no issues from EPIC, unless it's a GOG-specific issue. I'll wait for an update then, thanks!

**Comment by moraroy** [comment]
@denny-dee just updated the desktop version! stop and restart your scanner, let me know how it goes

**Comment by denny-dee** [comment]
Thanks, this works now! The Witcher appeared i games list and is running great! Dragon Age is uninstalled now due to storage, but I'm sure the problem is fixed. Thank you very much!

A few questions that I'm to lazy to find out myself, if you don't mind:
- If I gave the same game owned (and maybe even downloaded) from multiple sources, how will NSL handle it? I assume there will be duplicates, which is fine.
- What if I remove the Non-Steam game from Steam library and then download it again, will it be detected correctly?
- I have some Ubi games that I'd love to play through on the SD, but the launcher is killing me - it makes me log in every time. I have similar problem on my PC, which fixes itself sometimes. Or maybe I should just obtain these games from other sources?
- BTW, is it possible to simplify adding games to Steam library that I obtained from unsaid sources?
I just want to play Far Cry 3, Watch Dogs, and Prince of Persia Sand of Time without constantly entering my Ubisoft Connect password. And FC3 just won't start on SD, however, other sources contains all the fixes I need to play comfortably.
